scripts/repack-min-cardsets.bash

```diff
@@ -17,15 +17,15 @@ src_arc="$src_vbase.tar.bz2"
 
 if ! test -f "$src_arc"
 then
-    wget -c "https://sourceforge.net/projects/pysolfc/files/$src_base/$src_vbase/$src_arc/download" -O "$src_arc"
+    wget -c "https://pysolfc-cardsets-2-mirror.s3.eu-west-1.amazonaws.com/$src_base/$src_vbase/$src_arc" -O "$src_arc"
```

Comment:
I have created a mirror on a S3 bucket because `sourceforge.net` is very very slow.